CONTRIBUTING.md

```diff
@@ -35,20 +36,21 @@ grep -Irn NOTE: --exclude-dir=node_modules
 If you'd like your grammar to be included in this organization, please adhere to the following:
 
 1. Include corpus tests to ensure the parser works as expected.
-2. The external scanner (if any) must be written in C and not C++.
-3. The grammar must be written in JavaScript and not TypeScript or anything else.
-4. The queries must use the last-wins precedence, a-la Neovim style.
-    - Include highlight queries and their tests. Other queries are not necessary.
+    - You may also include some example files to be parsed in CI.
+2. The queries must use the last-wins precedence (all queries are applied).
+    - Include highlight queries and their tests. Other queries are generally not necessary.
     - The highlight captures should either use [Tree-sitter][STANDARD_CAPTURE_NAMES]'s
       "standard" names, or [Neovim][treesitter-highlight-groups]'s names.
-5. You should be somewhat active in maintaining it; if there's no need for maintenance
-   that's totally okay, but you shouldn't let issues pile up for months to years on end.
+3. The external scanner (if any) must be written in C and not C++.
```

Comment:
Call out the new `array` header?

Reply:
That should be in the official docs (which it is).